app/api/resources/grpc_client.py

```diff
@@ -48,18 +48,40 @@ def get_statistics(self, genome_uuid: str):
         toplevel_stats_by_uuid = self.stub.GetTopLevelStatisticsByUUID(genome_request)
         return toplevel_stats_by_uuid
 
-    def get_genome_details(self, genome_uuid: str, get_attributes: bool = True):
+    def get_genome_details(self, genome_uuid: str):
         # Create request
         request_class = self.reflector.message_class(
             "ensembl_metadata.GenomeUUIDRequest"
         )
-        request = request_class(genome_uuid=genome_uuid, get_attributes=get_attributes)
+        request = request_class(genome_uuid=genome_uuid)
 
         # Get response
         response = self.stub.GetGenomeByUUID(request)
 
         return response
 
+    def get_brief_genome_details(self, genome_uuid: str):
```

Comment:
The argument here is a string that may or may not be genome uuid.

Reply:
I renamed `genome_uuid` argument to `genome_uuid_or_slug` for clarification, I hope this is what you meant.